<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MitM</class>
 <widget class="QMainWindow" name="MitM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MitM</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>GSM-Netz erzeugen</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Administratorpasswort:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonWireshark">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Starten</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>OsmoBTS:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Wireshark:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Transceiver:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>OsmoBSC:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSipConnector">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Starten</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSQLBrowser">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>270</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Starten</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTransceiver">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Starten</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Sip-Connector:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBTS">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Starten</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBSC">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Starten</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>SQLiteBrowser:</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>pushButtonWireshark</zorder>
    <zorder>label_6</zorder>
    <zorder>label_4</zorder>
    <zorder>pushButtonSipConnector</zorder>
    <zorder>label_3</zorder>
    <zorder>label_2</zorder>
    <zorder>pushButtonSQLBrowser</zorder>
    <zorder>lineEdit</zorder>
    <zorder>pushButtonTransceiver</zorder>
    <zorder>label_5</zorder>
    <zorder>pushButtonBTS</zorder>
    <zorder>pushButtonBSC</zorder>
    <zorder>label_7</zorder>
    <zorder>label</zorder>
    <zorder>pushButtonWireshark</zorder>
    <zorder>label_4</zorder>
    <zorder>label_6</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>pushButtonSipConnector</zorder>
    <zorder>pushButtonSQLBrowser</zorder>
    <zorder>lineEdit</zorder>
    <zorder>pushButtonTransceiver</zorder>
    <zorder>label_5</zorder>
    <zorder>pushButtonBTS</zorder>
    <zorder>pushButtonBSC</zorder>
    <zorder>label_7</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Gespräch ausspähen</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PcapSipDump:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPcap">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Starten</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSlot1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Slot 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSlot2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Slot 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSlot3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Slot 3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSlot4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Slot 4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSlot5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Slot 5</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
